string name                         # object's name
uint8 pos_id                        # position id
--- 
